Objective-C/Tests/ReplicatorTest+CustomConflict.m

```diff
@@ -82,13 +84,19 @@ - (void) makeConflictFor: (NSString*)docID
     // Now make different changes in db and otherDB:
     doc1 = [[self.db documentWithID: docID] toMutable];
     [doc1 setData: localData];
-    Assert([self.db saveDocument: doc1 error: &error]);
     
     // pass the remote revision
     CBLMutableDocument* doc2 = [[otherDB documentWithID: docID] toMutable];
     Assert(doc2);
     [doc2 setData: remoteData];
-    Assert([otherDB saveDocument: doc2 error: &error]);
+    
+    if (localFirst) {
```

Comment:
What's this for? It shouldn't make any difference at all which order they're saved in, since they're in different databases. The revIDs and sequences are the same in either order.

Reply:
i am assuming if we save a document second, its revID will be greater than the previous(lexical order). hoping some kind of logic to link the revID with timestamp. 

My tests are working fine. unless it might fail in different machine.

Reply:
my assumptions were wrong, got it clarified now. Cool, I will re-write this.